ccloader/js/mod.js

```diff
@@ -300,18 +407,17 @@ export class Mod {
 	 * @param {string} dir 
 	 */
 	async _findAssets(dir){
-		if(window.isLocal || this.filemanager.isPacked(dir)){
+		if(window.isLocal || this.packed){
```

Comment:
`this.packed` is always `false` since `this.loaded` is set to `true` after this function has finished.